examples/example_automatic_termination_criterion.py

```diff
@@ -23,7 +23,10 @@
 try:
     import matplotlib.pyplot as plt
 except ImportError:
-    pass
+    print(
+        "This example requires matplotlib. You can install via pip install matplotlib"
+    )
+    raise
```

Comment:
What is the benefit of raising here if we have a try-catch? Isnt it better then to just remove it and import directly?

Reply:
makes sense